spec/setup_acceptance_node.pp

```diff
@@ -6,7 +6,8 @@
 
 class { 'candlepin::repo':
   version => pick(fact('candlepin_version'), 'nightly'),
-  baseurl => fact('candlepin_baseurl'),
+  #baseurl => fact('candlepin_baseurl'),
+  baseurl => "https://stagingyum.theforeman.org/candlepin/4.4/el${facts['os']['release']['major']}/x86_64/",
```

Comment:
You can override this fact by setting `BEAKER_FACTER_CANDLEPIN_BASEURL='...'`. You can also provide it in `.github/workflows/ci.yml`:
https://github.com/theforeman/foreman-installer-modulesync/blob/077114650945df080132f88e802383592e423758/moduleroot/.github/workflows/ci.yml.erb#L19-L20

We override it here:
https://github.com/theforeman/puppet-pulpcore/blob/738ca5a485fc4a71af1901de1260850349ad0035/.sync.yml#L3

Which gets implemented here:
https://github.com/theforeman/puppet-pulpcore/blob/738ca5a485fc4a71af1901de1260850349ad0035/.github/workflows/ci.yml#L18

Reply:
I don't understand this format with the `:` and the commas, and also how do you include variability? For example, since we need to vary the OS.

Reply:
I'll admit that I wrote it and still had to look it up and noticed I didn't have examples in https://github.com/voxpupuli/voxpupuli-acceptance/?tab=readme-ov-file#environment-variables-to-facts or in https://github.com/voxpupuli/gha-puppet. https://github.com/voxpupuli/puppet_metadata/commit/cba63c78141188d0966ea4db946433877887a2b7 at least documents that it's `FACT:LABEL:VALUES`.

It adds an additional vector to the matrix, so the existing OS & Puppet gets expanded to OS & Puppet & Fact.

Reply:
Right now I've set it back to the standard value as I think that's what we ultimately want this to be and not to be temporarily pointing at stage.